### PR TITLE
RFC8007bis draft13 extended status

### PR DESCRIPTION
**Pull request by gheist**

### CODE REVIEW

File: draft-ietf-cdni-ci-triggers-rfc8007bis.xml [hunk 1]
```diff
@@ -866,7 +874,22 @@
    for change, rather than repeatedly fetching the whole collection.  An
    example of this is given in <xref target="examples-polling-status-changes"/>.</t>
 
+   	<section title="Extended View Trigger Status Resource Collections" anchor="polling-collections-extended">
+	<t>
+		If dCDN advertises support for extended status, uCDN MAY request to provide extended 
+		Trigger Collection, that embeds all Trigger Status resources in the collection in the collection itsef,
```
Comment (jkrsvta) [inline draft-ietf-cdni-ci-triggers-rfc8007bis.xml]: ...uCDN MAY request _the_ extended Trigger Collection _view_, _which_ embeds all Trigger Status...
"in the collection" is repeated twice
itsef -> itself

File: draft-ietf-cdni-ci-triggers-rfc8007bis.xml [hunk 2]
```diff
@@ -866,7 +874,22 @@
...
 
+   	<section title="Extended View Trigger Status Resource Collections" anchor="polling-collections-extended">
+	<t>
+		If dCDN advertises support for extended status, uCDN MAY request to provide extended 
+		Trigger Collection, that embeds all Trigger Status resources in the collection in the collection itsef, 
+		in addition to a list of Trigger Status resource URLs. This support includes all types of 
+	   	Trigger Status Resources Collections, including filtered collections.
+	</t>	
+	<t>
+		uCDN SHOULD request the extended Trigger Collection view by passing query string parameter
```
Comment (jkrsvta) [inline draft-ietf-cdni-ci-triggers-rfc8007bis.xml]: passing _the_ query string parameter

File: draft-ietf-cdni-ci-triggers-rfc8007bis.xml [hunk 3]
```diff
@@ -866,7 +874,22 @@
...
+	<t>
+		If dCDN advertises support for extended status, uCDN MAY request to provide extended 
+		Trigger Collection, that embeds all Trigger Status resources in the collection in the collection itsef, 
+		in addition to a list of Trigger Status resource URLs. This support includes all types of 
+	   	Trigger Status Resources Collections, including filtered collections.
+	</t>	
+	<t>
+		uCDN SHOULD request the extended Trigger Collection view by passing query string parameter 
+		“status=extended” when requesting a Trigger Status Resource Collection.
+		dCDN SHOULD appropriately respond to the request for extended status – with HTTP Status 200 (“OK”)
```
Comment (jkrsvta) [inline draft-ietf-cdni-ci-triggers-rfc8007bis.xml]: The dash is unnecessary between status and with

File: draft-ietf-cdni-ci-triggers-rfc8007bis.xml [hunk 4]
```diff
@@ -1490,6 +1513,18 @@ uCDN-A                         dCDN-B                         dCDN-C
...
+	<t hangText="Name: all-triggers">
+	<vspace blankLines="1"/>
+	Description: Array of all Trigger Status objects in the collection.
+	Should be returned only when extended trigger collection view is requested as described in 
+	<xref target="polling-collections-extended"/>.
+	<vspace blankLines="1"/>
+	Value: An array of JSON-encoded Trigger Status objects.
+	<vspace blankLines="1"/>
+	Mandatory: No, only if dCDN supports and advertises the appropriate extended 
+	status for trigger collections.
```
Comment (jkrsvta) [inline draft-ietf-cdni-ci-triggers-rfc8007bis.xml]: I might be a bit confused by the wording.  So, this _is_ mandatory if the dCDN does not support and advertise the appropriate extended status for trigger collections?  Seems like it should be the opposite?

File: draft-ietf-cdni-ci-triggers-rfc8007bis.xml [hunk 5]
```diff
@@ -3473,6 +3508,61 @@ dCDN MAY reject such requests for urgent processing using “ereject” error co
...
+		CI/T Extended Trigger Status Capability object is used to indicate support for extended trigger status. 
+		The extended trigger status is returned upon uCDN request and includes:
+	</t>
+	<t>
+		<list style="symbols">
+		<t>"objects" attribute in Trigger Status object</t>	
+		<t>"objects" attribute in Error.v2 Description object</t>	
+		<t>"all-triggers" attribute in Trigger Collections object</t>	
+		</list>
+		The capability type is  "FCI.CITExtendedStatus".
```
Comment (jkrsvta) [inline draft-ietf-cdni-ci-triggers-rfc8007bis.xml]: Nit: extra space after "is"

File: draft-ietf-cdni-ci-triggers-rfc8007bis.xml (hunk 4 above)
Comment (gheist) [reply draft-ietf-cdni-ci-triggers-rfc8007bis.xml]: Fixed the wording.
+       Mandatory: No. The "all-triggers" SHOULD only be used by dCDN that supports and advertises 
+       the appropriate extended status for trigger collections (see <xref target="extended-status-capability-object"/> for detail

**Comment by jkrsvta** [comment]
I have completed my review of extended status and have no further comments.

**Comment by jkrsvta** [comment]
Merged with PR #8, so closing this PR.

**Pull request closed without merging**